deployment/helm/templates/ingress.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    # add an annotation indicating the issuer to use.
+    cert-manager.io/acme-challenge-type: http01
+    cert-manager.io/cluster-issuer: letsencrypt-prod
```

Comment:
shouldn't that `prod` label be some kind of variable so you can deploy staging/production easily too? or am I misunderstanding how things work?